harness-requirements.typ

```diff
@@ -44,30 +44,32 @@ Requirement priority is in ascending order, with P1 being the highest priority a
 };
 
 #let req(text, necessity, pri) = {
-	priority(pri) + [ ] + necessity + [ ]  + text + [ ] + "\n"
+	[
+		#figure(
+			priority(pri) + [ ] + necessity  + [ ] + "\n",
+			caption: text,
+			supplement: [Requirement],
+			kind: "req") <Requirement>
+	]
```

Comment:
`<Requirement>` here makes it so that every requirement has the label "Requirement". We need to be able to link to individual requirements.

```suggestion
	#figure(
		priority(pri) + [ ] + necessity  + [ ] + "\n",
		caption: text,
		supplement: [Requirement],
		kind: "req")
```
This will allow us to optionally specify a unique label for each requirement.